dmutils/content_loader.py

```diff
@@ -402,7 +402,10 @@ def _get_single_question_data(self, form_data):
                 "assurance": form_data.get(self.id + '--assurance'),
             }
 
-        return {self.id: value if value else None}
```

Comment:
We check that and return an empty list at the moment (https://github.com/alphagov/digitalmarketplace-utils/pull/224/files#diff-346b98f8c34fcc00d15a2cc1629cfb6aR384). I'm wondering if we should return `None` there too.